docs/make.jl

```diff
@@ -9,7 +9,8 @@ makedocs(
     authors  = "Joaquim Garcia",
     pages   = [
         "Home" => "index.md",
-        "manual.md"
+        "manual.md",
+        "theory.md"
```

Comment:
```suggestion
        "theory.md",
```
overall, it's common in Julia to leave a trailing comma, that way when you add a new line you don't need to modify the last one